.github/workflows/build-test.yaml

```diff
@@ -2567,27 +2570,21 @@ jobs:
             exit 1
           fi
 
-          if [ "$(kubectl get all -n postgres-test --ignore-not-found | wc -l | tr -d '\n')" != "0" ]; then
-            printf "postgres-test namespace still contains resources\n\n"
-            kubectl get all -n postgres-test
-            exit 1
-          fi
-
-          if [ "$(kubectl get all -n redis-test --ignore-not-found | wc -l | tr -d '\n')" != "0" ]; then
-            printf "redis-test namespace still contains resources\n\n"
-            kubectl get all -n redis-test
+          if [ "$(kubectl get all,secrets,configmap,pvc -A -l "kots.io/app-slug=$APP_SLUG" --ignore-not-found | wc -l | tr -d '\n')" != "0" ]; then
+            printf "Application resources are still found in the cluster\n\n"
+            kubectl get all,secrets,configmap,pvc -A -l "kots.io/app-slug=$APP_SLUG"
             exit 1
           fi
 
-          if [ "$(kubectl get all -n nginx-test --ignore-not-found | wc -l | tr -d '\n')" != "0" ]; then
-            printf "nginx-test namespace still contains resources\n\n"
-            kubectl get all -n nginx-test
+          if [ "$(kubectl get crd rabbitmqclusters.rabbitmq.com --ignore-not-found | wc -l | tr -d '\n')" != "0" ]; then
+            printf "rabbitmqclusters.rabbitmq.com CRD was not removed\n\n"
+            kubectl get crd rabbitmqclusters.rabbitmq.com
             exit 1
           fi
 
-          if [ "$(kubectl get pods -n "$APP_SLUG" -l "kots.io/app-slug=$APP_SLUG" --ignore-not-found | wc -l | tr -d '\n')" != "0" ]; then
-            printf "%s namespace still contains application resources\n\n" "$APP_SLUG"
-            kubectl get pods -n "$APP_SLUG" -l "kots.io/app-slug=$APP_SLUG"
+          if [ "$(kubectl get secret -A --field-selector type=helm.sh/release.v1 --ignore-not-found | wc -l | tr -d '\n')" != "0" ]; then
```

Comment:
Would probably be more accurate to iterate over the application namespaces here instead of using `-A` since we're not using the app slug label.

Reply:
addressed in https://github.com/replicatedhq/kots/pull/3853/commits/f4a3fe9835d8e3874d0f28e2c26a464ee24e2053